### ISSUE TITLE
Got 'dtype error' when using BatchNormalization layer.

### ISSUE THREAD

**Issue by dongb5** [fchollet/keras#5535]
Hello everyone! 

I have troubles when using BatchNormalization layer. I got a tensorflow error saying 'you must feed a value for placeholder tensor 'input_xx' with dtype float '. When I commented all BatchNormalization layers, the codes ran with no error. 

any suggestion please?
